### PR TITLE
refactor(node): apply optimizations

### PR DESCRIPTION
**Pull request by VoltrexKeyva**
Replaced unnecessary template strings and used less calls for optimization.

### CODE REVIEW

File: src/node/BrowserFetcher.ts [hunk 1]
```diff
@@ -112,10 +112,12 @@ function handleArm64(): void {
...
-          console.error(`The chromium binary is not available for arm64.`);
-          console.error(`If you are on Ubuntu, you can install with: `);
-          console.error(`\n sudo apt install chromium\n`);
-          console.error(`\n sudo apt install chromium-browser\n`);
+          console.error(
+            'The chromium binary is not available for arm64.' +
+              'If you are on Ubuntu, you can install with: ' +
+              '\n sudo apt install chromium\n' +
+              '\n sudo apt install chromium-browser\n'
+          );
```
Comment (mathiasbynens) [inline src/node/BrowserFetcher.ts:120]: This is not equivalent. Please revert this part of the change.

File: src/node/BrowserFetcher.ts [hunk 2]
```diff
@@ -528,9 +528,9 @@ function installDMG(dmgPath: string, folderPath: string): Promise<void> {
       mountPath = volumes[0];
       readdirAsync(mountPath)
         .then((fileNames) => {
-          const appName = fileNames.filter(
+          const appName = fileNames.find(
             (item) => typeof item === 'string' && item.endsWith('.app')
-          )[0];
+          );
```
Comment (mathiasbynens) [inline src/node/BrowserFetcher.ts:533]: Nice!

File: src/node/BrowserFetcher.ts (hunk 1 above)
Comment (VoltrexKeyva) [reply src/node/BrowserFetcher.ts:120]: It is equivalent now.

**Review by jschfflr — APPROVED** [review]
This change looks good to me, thanks!

**Pull request merged**